sentry/src/main/java/io/sentry/GsonSerializer.java

```diff
@@ -60,12 +60,12 @@
   /**
    * AndroidSerializer ctor
    *
-   * @param logger the ILogger interface
+   * @param options the SentryOptions object
    * @param envelopeReader the IEnvelopeReader interface
    */
   public GsonSerializer(
-      final @NotNull ILogger logger, final @NotNull IEnvelopeReader envelopeReader) {
-    this.logger = Objects.requireNonNull(logger, "The ILogger object is required.");
+      final @NotNull SentryOptions options, final @NotNull IEnvelopeReader envelopeReader) {
```

Comment:
`envelopeReader` is already a property from `SentryOptions`, we could remove it then

Reply:
👍